app.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+function StoreSalesGenerator(name, minCust, maxCust, avgSale) {
+  this.storeName = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgSale = avgSale;
+  this.numCookies = [];
+  this.totalSales = 0;
+  this.hour = 6;
+}
+
+var newText = document.createTextNode('text');
```

Comment:
This is never used in your code.  newText is re-assigned before used.  Consider moving this var declaration to the following line with the other declarations.